packages/bpk-component-map/src/BpkMap.js

```diff
@@ -114,27 +146,30 @@ const BpkMap = withGoogleMap((props: Props) => {
         streetViewControl: false,
         fullscreenControl: false,
         rotateControl: false,
+        clickableIcons: false,
```

Comment:
So, it's to make the icons unclickable, which means the infobox won't pop up when a icon is clicked
https://developers.google.com/maps/documentation/javascript/reference/map#MapOptions.clickableIcons
![image](https://user-images.githubusercontent.com/29422130/165707840-ba378216-747d-49f8-92ce-171e1967b883.png)